tb_plugin/test/test_compare_with_aurograd.py

```diff
@@ -0,0 +1,176 @@
+import os
+import time
+import unittest
+import torch
+import torch.nn as nn
+import torch.backends.cudnn as cudnn
+import torch.optim
+import torch.utils.data
+import torchvision
+import torchvision.transforms as T
+import torchvision.models as models
+from tensorboard_plugin_torch_profiler.profiler import RunLoader
+
+def create_log_dir():
+    log_dir_name='./log{}'.format(str(int(time.time()*1000)))
+    try:
+        os.makedirs(log_dir_name)
```

Comment:
Consider deleting the generated directories at the end of test, maybe in future when it is stable. For now, we may still need keep these for offline check.

Reply:
ok